clicache/integration-test/test.bat.in

```diff
@@ -30,7 +30,7 @@ set PATH=$<JOIN:$<SHELL_PATH:${PATH}>,;>;%PATH%
 set PATH=c:\Program Files (x86)\Nunit 2.6.4\bin;%PATH%
 
 set TEST_DIR="$<SHELL_PATH:${TEST_DIR}>"
-set GFCPP="%TEST_DIR%"
+set GEODE_NATIVE="%TEST_DIR%"
 
 set TESTSRC=$<SHELL_PATH:${CMAKE_CURRENT_SOURCE_DIR}>
 set GF_JAVA=$<SHELL_PATH:${Java_JAVA_EXECUTABLE}>
```

Comment:
There's more `GF` variable names, but as far as we could tell, they are not external facing to consumers.  So it might worth doing in another story to make this easier to review.